examples/vm-alpine-datavolume.yaml

```diff
@@ -20,7 +20,7 @@ spec:
         storageClassName: local
       source:
         http:
-          url: http://cdi-http-import-server.kubevirt/images/alpine.iso
+          url: registry:5000/kubevirt/alpine-container-disk-demo:devel
```

Comment:
does this work? I thought the source had to be docker or something and not http

Reply:
Right. A `docker://` prefix is needed. Fixed.